nginx/ts/ngx_stream_js_module.d.ts

```diff
@@ -66,7 +66,7 @@ interface NginxStreamVariables {
     readonly 'time_iso8601'?: NjsByteString;
     readonly 'time_local'?: NjsByteString;
 
-    [prop: string]: NjsByteString;
+    [prop: string]: NjsByteString | undefined;
```

Comment:
@jirutka Can you share the exact code snippet that causes the issue?

Reply:
Sure, it’s part of #344:

```sh
git clone -b njs-types-pkg git@github.com:jirutka/njs.git
cd njs/ts
npm install
npm test
```

The problem is that you’re running `tsc` with the defaults – backward compatible settings that are very lax and shouldn’t be used for any new projects.

Reply:
Sorry for mixing this commit into this PR, I should have created a new PR for it.

Reply:
I’m gonna move this commit into #344 and close this PR, ’cause it has been already merged, except this one.